src/components/structures/RoomView.tsx

```diff
@@ -83,6 +83,7 @@ import { objectHasDiff } from "../../utils/objects";
 import SpaceRoomView from "./SpaceRoomView";
 import { IOpts } from "../../createRoom";
 import {replaceableComponent} from "../../utils/replaceableComponent";
+import _ from 'lodash';
```

Comment:
we prefer the `import { fn }` syntax in hopes that treeshaking does the right thing (even though it doesn't).